app/views/insured/family_members/_dependent_form.html.erb

```diff
@@ -154,7 +154,11 @@
           <div class="col-md-5 mt18">
             <div class="select-positioning ml-label"> <label>is</label> </div>
             <label class="static_label label-floatlabel mt-label">RELATIONSHIP |
-              <a href="#relationship" data-toggle="modal" data-target="#relationship">Not sure?</a>
+              <% if FinancialAssistanceRegistry[:enroll_app].settings(:site_key).item.to_s.downcase == 'me' %>
+                <a href="#relationship" data-toggle="modal" data-target="#relationship">Not sure?</a>
+              <% else %>
+                <a class='hidden'></a>
```

Comment:
These are so we don't mess up the x code for the cucumbers

They don't appear visible on the view

I'm desperate